crates/nu-command/src/dataframe/values/nu_dataframe/between_values.rs

```diff
@@ -366,7 +366,8 @@ pub(super) fn compute_series_single_value(
                 rhs_span: right.span()?,
             }),
         },
-        Operator::Contains => match &right {
+        // TODO: update this to do a regex match instead of a simple contains?
+        Operator::RegexMatch => match &right {
```

Comment:
@elferherrera I've updated this already; is there another place I forgot? It builds successfully locally with `cargo build --features=extra`

Reply:
That's all. It is the only place where that is used